src/load_inputs/load_fuels_data.jl

```diff
@@ -32,9 +31,11 @@ function load_fuels_data!(setup::Dict, path::AbstractString, inputs::Dict)
     scale_factor = setup["ParameterScale"] == 1 ? ModelScalingFactor : 1
 
     for i = 1:length(fuels)
+            # fuel cost is in $/MMBTU w/o scaling, $/Billon BTU w/ scaling
             fuel_costs[fuels[i]] = costs[:,i] / scale_factor
-            # fuel_CO2 is kton/MMBTU with scaling, or ton/MMBTU without scaling.
-            fuel_CO2[fuels[i]] = CO2_content[i] / scale_factor
+            # fuel_CO2 is ton/MMBTU w/o scaling, or kton/Billion BTU w/ scaling.
+            # No need to scale fuel_CO2
```

Comment:
The previous person was not wrong, but he/she basically scaled it twice, by dividing 1000 in load_fuels_data and then multiplying by 1000 in load_generator_data/jl

If you look at the https://github.com/GenXProject/GenX/blob/develop/src/load_inputs/load_generators_data.jl, line 208.
`gen_in[g,:CO2_per_MWh] *= scale_factor
`
because the previous one scaled the fuel_co2 to ton/Billion BTU, they need to multiply 1000 to get ton/Gwh to kton/GWh.

If they didn't scale the fuel_co2, the fuel_co2 should be ton/MWh or kton/GWh., then we don't need line 208. 

Also, after introducing co2.jl and fuel.jl, we don't need to determine CO2_per_MWh because they are tracked by fuel consumption.